www/docs/client/links/retryLink.md

```diff
@@ -0,0 +1,79 @@
+---
+id: retryLink
+title: Retry Link
+sidebar_label: Retry Link
+slug: /client/links/retryLink
+---
+
+`retryLink` is a link that allows you to retry failed operations in your tRPC client. It provides a customizable way to handle transient errors, such as network failures or server errors, by automatically retrying the failed requests based on specified conditions.
+
+:::tip
+If you use `@trpc/react-query` you don't generally need this as it's built into `@tanstack/react-query`.
```

Comment:
might be ambiguous, not everything in @trpc/react-query is built on react query and have it baked in, useSubscription being the subject of this PR. Not sure how to add clarity without getting wordy though